src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -166,7 +166,7 @@ static void usage()
             << "  --read-only                   Map read-only\n"
             << "  --reattach-timeout <sec>      Set nbd re-attach timeout\n"
             << "                                (default: " << Config().reattach_timeout << ")\n"
-            << "  --try-netlink                 Use the nbd netlink interface\n"
+            << "  --try-netlink                 Use the nbd netlink interface (deprecated)\n"
```

Comment:
We should probably remove the option from usage entirely, but continue to accept it in the parser.  I can see "deprecated" here being interpreted as "nbd netlink interface is deprecated" as opposed to just the option itself.

Reply:
Good point! Removed